src/Parser/Mage/Fire/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,178 @@
+      requirements: () => {
+        return [
+          new Requirement({
+            name: "Fire Blast used without Heating Up",
+            check: () => this.heatingUp.fireBlastUtilSuggestionThresholds,
+          }),
+          new Requirement({
+            name: "Phoenix's Flames used without Heating Up",
+            check: () => this.heatingUp.phoenixFlamesUtilSuggestionThresholds,
+          }),
+          new Requirement({
+            name: "Hard Cast Before Hot Streak Util",
+            check: () => this.hotStreak.castBeforeHotStreakThresholds,
+          }),
+          new Requirement({
+            name: "Expired Hot Streaks",
+            check: () => this.hotStreak.expiredProcsThresholds,
+          }),
+          new Requirement({
+            name: "Wasted Crits",
+            check: () => this.hotStreak.wastedCritsThresholds,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Managing Combustion',
+      description: <Wrapper><SpellLink id={SPELLS.COMBUSTION.id}/> is the only major cooldown that Fire Mage has and it plays a massive role in your performance. When Combustion is about to come off cooldown, it is important that you bank a couple charges of <SpellLink id={SPELLS.FIRE_BLAST.id}/> and <SpellLink id={SPELLS.PHOENIXS_FLAMES.id}/> to help you get as many Hot Streak procs as possible during Combustion. Additionally, during Combustion, your goal is to use your instant cast spells to convert Heating Up to Hot Streak as quickly as possible so you can get more instant cast Pyroblasts in during the Combustion Window. You should only hard cast spells like <SpellLink id={SPELLS.FIREBALL.id}/> and <SpellLink id={SPELLS.SCORCH.id}/> if you are completely out of instant cast spells. Finally, if you have <ItemLink id={ITEMS.MARQUEE_BINDINGS_OF_THE_SUN_KING.id}/>, you should make sure you are using your procs to hard cast <SpellLink id={SPELLS.PYROBLAST.id}/> during Combustion as long as you can finish the cast before Combustion ends.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: "Combustion Used with < 2 Phoenix Flames Charges",
+            check: () => this.combustionCharges.phoenixFlamesThresholds,
+          }),
+          new Requirement({
+            name: `Combustion Used with < ${this.combatants.selected.hasTalent(SPELLS.FLAME_ON_TALENT.id) ? 2 : 1} Fire Blast Charges`,
```

Comment:
If possible, I think it would make more sense to show how many Fire Blasts were actually cast per combustion, rather than the number available at the start of it.